#Do not check netCDF LD_LIBRARY_PATH dependencies using nc-config --libs, as we provide with EB
#B. Hajgato August 20th, 2015
--- gmt-5.1.2/cmake/modules/FindNETCDF.cmake.old	2014-02-01 00:39:56.000000000 +0100
+++ gmt-5.1.2/cmake/modules/FindNETCDF.cmake	2015-08-20 11:06:25.039494965 +0200
@@ -106,12 +106,12 @@
 )
 
 # find all libs that nc-config reports
-foreach (_extralib ${_netcdf_lib})
-	find_library (_found_lib_${_extralib}
-		NAMES ${_extralib}
-		PATHS ${_netcdf_libpath})
-	list (APPEND NETCDF_LIBRARY ${_found_lib_${_extralib}})
-endforeach (_extralib)
+#foreach (_extralib ${_netcdf_lib})
+#	find_library (_found_lib_${_extralib}
+#		NAMES ${_extralib}
+#		PATHS ${_netcdf_libpath})
+#	list (APPEND NETCDF_LIBRARY ${_found_lib_${_extralib}})
+#endforeach (_extralib)
 
 if (NETCDF_LIBRARY AND NETCDF_INCLUDE_DIR AND NOT HAVE_NETCDF4)
   # Ensure that NetCDF with version 4 extensions is installed
